requirements/edx/base.txt

```diff
@@ -55,7 +55,7 @@ edx-django-sites-extensions==2.1.1
 edx-enterprise==0.33.0
 edx-oauth2-provider==1.2.0
 edx-opaque-keys==0.4.0
-edx-organizations==0.4.3
+edx-organizations==0.4.4
```

Comment:
It's related. See https://github.com/edx/edx-organizations/compare/0.4.3...0.4.4.